README.md

```diff
@@ -1731,6 +1732,12 @@ _See also: [awesome-sysadmin/Code Review](https://github.com/awesome-foss/awesom
 - [UVDesk](https://www.uvdesk.com/) - UVDesk community is a service oriented, event driven extensible opensource helpdesk system that can be used by your organization to provide efficient support to your clients effortlessly whichever way you imagine. ([Demo](https://demo.uvdesk.com/), [Source Code](https://github.com/uvdesk/community-skeleton)) `MIT` `PHP`
 - [ZenTao](https://www.zentao.pm/) - An agile(scrum) project management system/tool. ([Demo](https://demo.zentao.pm/user-login.html), [Source Code](https://github.com/easysoft/zentaopms)) `ZPL-1.2` `PHP`
 
+### Software Development - Security & QA
+
+**[`^        back to top        ^`](#)**
+
+- [Scanmycode CE (Community Edtion)](https://www.scanmycode.today/) - Code Scanning/SAST/Static Analysis/Linting using many tools/Scanners with One Report - Scanmycode Community Edition (CE). ([Source Code](https://github.com/marcinguy/scanmycode-ce)) `BSD-3-Clause LGPL-2.1 Commonsclause` `Python`
```

Comment:
Hi,

https://github.com/awesome-selfhosted/awesome-selfhosted/blob/master/.github/PULL_REQUEST_TEMPLATE.md

> Any category you are creating has the minimum requirement of 3 items.
>If not, your addition may be inserted into Misc/Other.

Reply:
@nodiscc OK, sorry, somehow overlooked it.

Added 2 other most popular items I know that should fit in the category.